docs/about/compatibility/precision-support.rst

```diff
@@ -1,13 +1,14 @@
 .. meta::
   :description: Supported data types in ROCm
-  :keywords: int8, float8, float8 (E4M3), float8 (E5M2), bfloat8, float16, half, bfloat16, tensorfloat32, float, float32, float64, double, AMD, ROCm, AMDGPU
-
-.. _rocm-supported-data-types:
+  :keywords: int8, float8, float8 (E4M3), float8 (E5M2), bfloat8, float16, half, bfloat16, tensorfloat32, float,
+   float32, float64, double, AMD, ROCm, AMDGPU
 
 *************************************************************
-ROCm data type specifications
+Precision support
 *************************************************************
 
+Use the following sections to identify which data types and HIP types are supported with ROCm.
```

Comment:
Use the following sections to identify data types and HIP types ROCm supports.

Reply:
fixed